lib/bundler_api/web.rb

```diff
@@ -227,6 +227,14 @@ def version_line(row)
     line = row[:number].to_s
     line << "-#{row[:platform]}" unless row[:platform] == "ruby"
     line << " " << deps.join(",") if deps.any?
+
+    required = []
+    ruby_version = row[:required_ruby_version]
+    required << "#{row[:platform]}:#{ruby_version}" if ruby_version
```

Comment:
Here, should I use platform? `Gem::Specification#required_ruby_version` don't give me a platform, it actually returns a `Gem::Requirement` which only contains something like `">1.1.1"`